src/cms/templates/pagination.html

```diff
@@ -37,6 +38,16 @@
             <i data-feather="chevron-right"></i>
         </a>
     </div>
+    <div>
+        <label for="chunk-size"></label>
+        <select id="chunk-size">
+            <option value="">{% trans 'Number of entries per page' %}</option>
+            <option value="{{ url }}?size=10" {% if request.GET.size == "10" %}selected{% endif %}>10</option>
+            <option value="{{ url }}?size=20" {% if request.GET.size == "20" %}selected{% endif %}>20</option>
+            <option value="{{ url }}?size=50" {% if request.GET.size == "50" %}selected{% endif %}>50</option>
+            <option value="{{ url }}?size=100" {% if request.GET.size == "100" %}selected{% endif %}>100</option>
+        </select>
+    </div>
```

Comment:
> Maybe you could show the selection input all the time, even if there are less entries than the current size (so only one page)? This way, it would be possible to e.g. go back from selecting 100 entries to less...

Flex direction and chunk size visibility is done altogether in 6745f7d